mockgen/mockgen.go

```diff
@@ -622,22 +628,23 @@ func (g *generator) Output() []byte {
 	return src
 }
 
-func lookupPackageName(importPath string) (string, bool) {
-	var pkg struct {
-		Name string
-	}
+func lookupPackagesName(importPaths []string) map[string]string {
```

Comment:
I think the name of this method should change. Maybe `createPackageMap`?

Reply:
ok, renamed to createPackageMap